scss/functions/_get-color.scss

```diff
@@ -7,4 +7,13 @@
   // if they are valid, return the color
   // if they are not valid warn 'Invalid color type or level' and return null
   // null is a special value in Sass that means nothing, and it is used to represent the absence of a value, you need to return null if the type or level is not valid
+
+  $color-map: map.get($colors, $type);
+
+  @if $color-map != null and map.has-key($color-map, $level) {
```

Comment:
masih kurang tepat ya
pada line 13 `$color-map != null` pada masih kurang tepat
jadi kamu tinggal melengkapi saja titik disini
`@if .... and map.has-key($color-map, $level)`